clients/src/test/java/org/apache/kafka/common/protocol/ApiKeysTest.java

```diff
@@ -55,11 +61,14 @@ public void schemaVersionOutOfRange() {
      */
     @Test
     public void testResponseThrottleTime() {
-        List<ApiKeys> authenticationKeys = Arrays.asList(ApiKeys.SASL_HANDSHAKE, ApiKeys.SASL_AUTHENTICATE);
+        Set<ApiKeys> authenticationKeys = EnumSet.of(ApiKeys.SASL_HANDSHAKE, ApiKeys.SASL_AUTHENTICATE);
+        // Newer protocol apis include throttle time ms even for cluster actions
+        Set<ApiKeys> clusterActionsWithThrottleTimeMs = EnumSet.of(ApiKeys.ALTER_ISR);
```

Comment:
We have to decide whether we want to do this or remove the throttle time ms from AlterIsr before 2.7.0.

Reply:
cc @bbejeck

Reply:
We decided this is not a blocker.